dependency-check-core/src/main/java/org/owasp/dependencycheck/analyzer/JarAnalyzer.java

```diff
@@ -747,6 +741,9 @@ protected boolean parseManifest(Dependency dependency, List<ClassNameInformation
                 foundSomething = true;
                 versionEvidence.addEvidence(source, "specification-version", specificationVersion, Confidence.HIGH);
             }
+        } catch (IOException ex) {
+            LOGGER.warn("Unable to read JarFile '{}'.", dependency.getActualFilePath());
```

Comment:
Minor: maybe update the message to read something like `Unable to read dependency file '{}'.` as the {{JarAnalyzer}} can also handle {{war}} files.